popup.html

```diff
@@ -62,7 +62,8 @@ <h1>
           <button id="sendtocb" class="slds-button_stretch slds-input_height" data-locale="TT_SendTo">Send To</button>
         </div>
         <div class="slds-col slds-size_2-of-3">
-          <select id="sendto" class="slds-select" ></select>
```

Comment:
avoid this, changes that have no semantic value are noise for the person who has to review it (me) and usually makes it harder to maintain. SInce you have various of these in this commit. Please delete the PR and make a new one that contains ONLY the necessary changes